########################################
# Sensor
########################################

uint32 type      # Sensor type (e.g. for sensor: image, audio),
                 # refer to rtabmap::Sensor::Type).
rtabmap/CvMatMsg matrix
                 
